ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -873,16 +878,16 @@ public void replicaThreadTest() throws Exception {
     currentTimeMs = time.milliseconds();
     replicaThread.replicate(new ArrayList<>(replicasToReplicate.values()));
     assertMissingKeys(missingKeys, batchSize, replicaThread, remoteHost, replicasToReplicate);
-    assertEquals(
-        "Replica thread should sleep exactly replication.thread.throttle.sleep.duration.ms since remote has new token",
-        currentTimeMs + config.replicationReplicaThreadThrottleSleepDurationMs, time.milliseconds());
+    assertEquals("Replica thread should sleep exactly " + expectedThrottleDurationMs + " since remote has new token",
+        currentTimeMs + expectedThrottleDurationMs, time.milliseconds());
 
     // verify that throttling on the replica thread is disabled when replication.thread.throttle.sleep.duration.ms is 0.
     Properties properties = new Properties();
     properties.setProperty("replication.thread.throttle.sleep.duration.ms", "0");
```

Comment:
this property is supposed to be changed for either `intra` or `inter` sleep duration.

Reply:
Fixed